static_content/_site/support/thank-you/index.html

```diff
@@ -622,10 +622,10 @@ <h2 class="modal__title">Quick search</h2>
               
               
 
-              <h2 class="promo__heading">Plagiarism checking</h2>
+              <h2 class="promo__heading">Provisional ISSNs versus confirmed ISSNs</h2>
```

Comment:
I think we don't generate static content before PR anymore, these will often conflict. Generate on merge